### PR TITLE
GCE/Windows: add instructions about stackdriver logging in README

### PR DESCRIPTION
**Pull request by yujuhong**
Setting LOGGING_STACKDRIVER_RESOURCE_TYPES ensures that the both Linux
and Windows nodes send logs to the same resources with the same tags.

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design

/kind documentation

> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by mtaufen** [comment]
/lgtm

Looks like the PR description has some uncommented template stuff.

File: cluster/gce/windows/README-GCE-Windows-kube-up.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ plane only runs on Linux.
 export NUM_NODES=2  # number of Linux nodes
 export NUM_WINDOWS_NODES=2
 export KUBE_GCE_ENABLE_IP_ALIASES=true
+export LOGGING_STACKDRIVER_RESOURCE_TYPES=new
```
Comment (pjh) [inline cluster/gce/windows/README-GCE-Windows-kube-up.md]: Should we set this for our test jobs too?

(I recognize it's a different repo so it wouldn't be in this PR.)

Comment (yujuhong) [reply cluster/gce/windows/README-GCE-Windows-kube-up.md]: Yep. We should set it our test configurations.

**Pull request merged**